deps/wazuh_testing/wazuh_testing/scripts/generate_fim_events.py

```diff
@@ -0,0 +1,173 @@
+
+            return key[0]  # Ignore the flag that RegCreateKeyEx returns
+        except OSError as e:
+            print(f"Registry could not be created: {e}")
+        except pywintypes.error as e:
+            print(f"Registry could not be created: {e}")
+
+
+def delete_registry(key, subkey, arch):
+    """Delete a registry key.
+
+    Args:
+        key (pyHKEY): the key of the registry (HKEY_* constants).
+        subkey (str): the subkey (name) of the registry.
+        arch (int): architecture of the registry (KEY_WOW64_32KEY or KEY_WOW64_64KEY).
+    """
+    if sys.platform == 'win32':
+        print_arch = '[x64]' if arch == KEY_WOW64_64KEY else '[x32]'
+        print(f"Removing registry key {print_arch}{str(os.path.join(registry_class_name[key], subkey))}")
+
+        try:
+            key_h = win32api.RegOpenKeyEx(key, subkey, 0, win32con.KEY_ALL_ACCESS | arch)
+            win32api.RegDeleteTree(key_h, None)
+            win32api.RegDeleteKeyEx(key, subkey, samDesired=arch)
+        except OSError as e:
+            print(f"Couldn't remove registry key {str(os.path.join(registry_class_name[key], subkey))}: {e}")
+        except pywintypes.error as e:
+            print(f"Couldn't remove registry key {str(os.path.join(registry_class_name[key], subkey))}: {e}")
+
+
+def modify_registry_value(key_h, value_name, type, value):
+    """
+    Modify the content of a registry. If the value doesn't not exists, it will be created.
+
+    Args:
+        key_h (pyHKEY): the key handle of the registry.
+        value_name (str): the value to be set.
+        type (int): type of the value.
+        value (str): the content that will be written to the registry value.
+    """
+    if sys.platform == 'win32':
+        try:
+            print(f"Modifying value '{value_name}' of type {registry_value_type[type]} and value '{value}'")
+            win32api.RegSetValueEx(key_h, value_name, 0, type, value)
+        except OSError as e:
+            print(f"Could not modify registry value content: {e}")
+        except pywintypes.error as e:
+            print(f"Could not modify registry value content: {e}")
+
+
+def generate_events(test_files, file_size, eps):
+    generated_events = 0
+    n_events = int(eps/len(test_files))
+    remain_events = eps % len(test_files)
+    for _ in range(n_events):
+        if sys.platform == 'win32':
+            random_string = ''.join(random.choice(string.ascii_letters) for _ in range(10))
+            for n_registry in range(1, len(test_files)+1):
+                key_h = win32api.RegOpenKeyEx(registry_parser[KEY], f'{testreg}{n_registry}', 0, KEY_ALL_ACCESS | KEY_WOW64_64KEY)
+                modify_registry_value(key_h, reg_value, REG_SZ, random_string)
+                generated_events += 1
+        else:
+            random_string = ''.join(random.choice(string.ascii_letters) for _ in range(file_size))
+            print(random_string)
```

Comment:
```suggestion
```

Reply:
Done in [804fb74](https://github.com/wazuh/wazuh-qa/pull/4972/commits/804fb74d4e757a727d519e5e32c8cdf833567aa0)